src/schema-parser/asyncapi-schema-parser.ts

```diff
@@ -9,6 +7,13 @@ import type { AsyncAPISchema, SchemaValidateResult } from '../types';
 import type { SchemaParser, ParseSchemaInput, ValidateSchemaInput } from '../schema-parser';
 import type { v2 } from '../spec-types';
 
+// import only 2.X.X AsyncAPI JSON Schemas for better treeshaking
+import * as asyncAPI2_0_0Schema from '@asyncapi/specs/schemas/2.0.0.json';
+import * as asyncAPI2_1_0Schema from '@asyncapi/specs/schemas/2.1.0.json';
+import * as asyncAPI2_2_0Schema from '@asyncapi/specs/schemas/2.2.0.json';
+import * as asyncAPI2_3_0Schema from '@asyncapi/specs/schemas/2.3.0.json';
+import * as asyncAPI2_4_0Schema from '@asyncapi/specs/schemas/2.4.0.json';
```

Comment:
This is not scalable at all. I don't want to keep adding lines here every time we release a version.